common/predictive-text/package.json

```diff
@@ -1,7 +1,28 @@
 {
   "name": "@keymanapp/lexical-model-layer",
   "description": "Keyman/Predictive Text integration layer",
-  "main": "build/index.js",
+  "main": "build/obj/node/index.js",
+  "exports": {
+    ".": {
+      "node": {
+        "types": "./build/obj/node/index.d.ts",
+        "import": "./build/obj/index.js"
+      },
+      "default": {
+        "types": "./build/obj/web/index.d.ts",
+        "import": "./build/obj/web/index.js"
+      }
+    },
+    "./node": "./build/obj/node/index.js",
+    "./web": "./build/obj/web/index.js",
+    "./build/obj/node/index.js": "./build/obj/node/index.js",
+    "./build/obj/web/index.js": "./build/obj/web/index.js"
```

Comment:
Our current version of TypeScript does not know how to read from the export mapping, so we need an unaliased export version for now to use in the source and when compiling.

Reply:
Which version of TS is this resolved in?

Reply:
TS 4.7.